$NetBSD: patch-Makefile,v 1.1 2016/07/24 11:48:14 wiz Exp $

Add DESTDIR support.
Support install(1) that does not create directories automatically.

--- Makefile.orig	2015-12-03 16:45:59.000000000 +0000
+++ Makefile
@@ -6,6 +6,7 @@
 
 # installation prefix
 PREFIX=/usr/local
+DESTDIR?=
 
 ########################################
 #
@@ -61,8 +62,10 @@ run: $(TARGET)
 	./$(TARGET)
 
 install: $(TARGET)
-	$(INSTALL) -s -D $(TARGET) $(PREFIX)/bin/$(TARGET)
-	$(INSTALL) -D einstein.res $(PREFIX)/share/einstein/res/einstein.res
+	$(INSTALL) -d $(DESTDIR)$(PREFIX)/bin
+	$(INSTALL) -d $(DESTDIR)$(PREFIX)/share/einstein/res
+	$(INSTALL) -s $(TARGET) $(DESTDIR)$(PREFIX)/bin/$(TARGET)
+	$(INSTALL) einstein.res $(DESTDIR)$(PREFIX)/share/einstein/res/einstein.res
 	
 # DO NOT DELETE THIS LINE -- make depend depends on it.
 
